blocks/top-table-list-block/themes/news.json

```diff
@@ -366,15 +387,18 @@
 			"default": {
 				"components": {
 					"heading": {
-						"align-self": "left",
-						"justify-content": "left"
+						"align-self": "inline-start",
+						"justify-content": "inline-start"
 					},
 					"overline": {
 						"display": "flex",
-						"justify-content": "left"
+						"justify-content": "inline-start"
```

Comment:
Value is just `start`.

Reply:
@vgalatro Updated [4bf7fbb](https://github.com/WPMedia/arc-themes-blocks/pull/1830/commits/4bf7fbb991a802ef23fdbcf606d6de0e075127e5)